source/extensions/tracers/datadog/time_util.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include <datadog/clock.h>
+
+#include "envoy/common/time.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace Tracers {
+namespace Datadog {
+
+// Return the `datadog::tracing::TimePoint` that most closely matches the
+// specified `SystemTime`. Use the optionally specified
+// `datadog::tracing::Clock` to read the current time. If a clock is not
+// specified, then the default clock is used.
```

Comment:
can you format these using doxygen syntax per other envoy files?

Reply:
Sure, I'll make that change here and elsewhere.